docs/design-code-status.njk

```diff
@@ -1,302 +1,117 @@
 ---
 layout: layout-basic.njk
 title: Design/code status
+stylesheets:
+  - /assets/packages/@rhds/elements/elements/rh-table/rh-table-lightdom.css
+importElements:
+  - rh-tag
+  - rh-table
 ---
 
 
 {% import 'component/components.njk' as components %}
 
 {% call components.section("Overview") -%}
 
-  <p>Our team is building a robust library of foundations and components. When we add new foundations and components or make significant updates, this table will be updated so you know what is currently available to use.</p>
+  <p>A detailed synopsis of our web components and their implementation status.</p>
+  <rh-table class="component-status-table">
```

Comment:
@bennypowers this is just one off table to be only used in this template. Its kind of a legend table for various statuses.